charts/prometheus-elasticsearch-exporter/values.yaml

```diff
@@ -18,13 +18,25 @@ image:
   pullPolicy: IfNotPresent
   pullSecret: ""
 
-## Set enabled to false if you don't want securityContext
-## in your Deployment.
-## The below values are the default for kubernetes.
-## Openshift won't deploy with runAsUser: 1000 without additional permissions.
-securityContext:
-  enabled: true  # Should be set to false when running on OpenShift
+## Set to `null` to keep pod level securityContext unspecified
+#podSecurityContext: null
+podSecurityContext:
+  runAsNonRoot: true
+  ## Openshift won't deploy with runAsUser: 1000 without additional permissions.
+  ## Remove the field by setting it to `null`:
+  #runAsUser: null
```

Comment:
why adding it at all if its disabled?
imho only the current defaults should be used again.

Reply:
This was meant to demonstrate how to unset the field, e.g. that there must not be quotes around the `null`.

Reply:
to unset your normal just use it like:
```
podSecurityContext: {}
```

Reply:
I thought so, too, and tested it but this produces the following for me:

```
      securityContext:
        - ""
```